demos/makefile.cases.inc

```diff
@@ -17,11 +17,11 @@ gia_demos := $(addprefix $(gia_dir)/,$(gia_demo_names))
 gia_cases := $(addprefix $(gia_dir)/,$(gia_test_names))
 
 # other cases not tied to specific physics/domains
-extra_cases := PDE_constrained_optimisation
+extra_cases := PDE_constrained_optimisation smoothing
```

Comment:
Should we create a directory called `extras` to group these demos not tied to the main G-ADOPT categories?